README.md

```diff
@@ -103,6 +105,26 @@ See the documentation for
 and
 [MegaLinter configuration](https://oxsecurity.github.io/megalinter/latest/configuration/).
 
+### `megalinter-incremental`
+
+Run [MegaLinter](https://oxsecurity.github.io/megalinter/) (skipping linters
+that run in project mode) by running:
+
+```sh
+npx -- mega-linter-runner@<version> \
+  --containername megalinter-$(basename "$PWD") \
+  --remove-container \
+  --fix \
+  --env CLEAR_REPORT_FOLDER=true \
+  --env LOG_LEVEL=warning \
+  --filesonly
+```
+
+See the documentation for
+[`mega-linter-runner`](https://oxsecurity.github.io/megalinter/latest/mega-linter-runner/#usage)
+and
+[MegaLinter configuration](https://oxsecurity.github.io/megalinter/latest/configuration/).
```

Comment:
Multiple instances: it would be good to clarify in the documentation of all four MegaLinter hooks whether it is intended for MegaLinter v5 or v6 since otherwise the user has to do extra research to sort out why we have four hooks at this release and which suits their use case.